### PR TITLE
Stop validating certgen/templates

### PR DESCRIPTION
**Pull request by kargakis**
/assign jim-minter 

Fixes https://github.com/kargakis/acs-engine/issues/89

### CODE REVIEW

**Comment by kargakis** [comment]
/hold cancel

and 

/test 

for the sake of it

File: pkg/certgen/helpers.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package certgen
```
Comment (jim-minter) [inline pkg/certgen/helpers.go]: could you rename this file, e.g. to novalidate.go, and add a comment about why it's here?

**Comment by kargakis** [comment]
@jim-minter I think it will but not 100% sure. Seems like the flake occurs very often so we should know whether it's fixed or not soon.

File: pkg/certgen/helpers.go (hunk 1 above)
Comment (kargakis) [reply pkg/certgen/helpers.go]: done

**Comment by jim-minter** [comment]
cool
/lgtm

**Pull request merged**